Apply-For-Me-UI/src/pages/contact_us/components/ContactStyle.css

```diff
@@ -0,0 +1,467 @@
+        display: flex;
+        flex-direction: column;
+
+    }
+    .top-tier .icon, .low-tier .icon{
+        margin-bottom: 10px;
+        width: 25px;
+        height: 25px;
+    }
+    .stripe{
+        position: absolute;
+        left: 0;
+        top: 0;
+        padding: 20px;
+        padding-left: 0;
+        height: 100vh;
+        background-color: white;
+        box-shadow: 1px 1px 1px lightgray;
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+    }
+}
+/* form styling */
+
+.form-group{
+    display: flex;
+    flex-direction: column;
+    width: inherit;
+}
+.form-group input{
+    margin-top: 5px;
+    margin-bottom: 20px;
+    border: 1px solid lightgray;
+    border-radius: 5px;
+    padding: 10px;
+}
+
+.form-first-line{
+    display: flex;
+    
+}
+.first-line-1 ,.first-line-2{
+    flex: 1;
+
+
+}
+.first-line-2{
+    margin-left: 20px;
+}
+.form-fifth-line{
+    margin-bottom: 20px;
+    margin-top: 20px;
+}
+
+.privacy{
+    margin-left: 5px;
+}
+.radio-group{
+    display: flex;
+}
+input.radio{
+    transform: scale(3);
+    padding: 5px;
+}
+.form button{
+    padding: 10px;
+    text-align: center;
+    border-radius: 5px;
+    background-color: #2E3192;
+    color: white;
+    width: 100%;
+    border: none;
+}
+label{
+    font-weight: 600;
+}
+textarea{
+    
+    font-size: 1em;
+    background-color: white;
+    resize:none;
+    
+    width: 100%;
+    border: none;
+}
+textarea:focus{
+    outline: none;
+  
+}
+.textarea{
+    width: inherit;
+   font-family: montserrat-medium;
+    padding: 10px;
+    border: 1px solid lightgray;
+    border-radius: 5px;
+    background-color: white;
+    height: 300px;
+    margin-top: 5px;
+}
+.contact-mid{
+    width: 100%;
+}
+@media screen and (max-width:768px) {
+    .first-line-2{
+        margin-left: 0;
+    }
+    .form-first-line{
+        display: flex;
+        flex-wrap: wrap;
+        
+        
+    }
+    
+}
+
+@media screen and (max-width:481px) {
+
+ .form-first-line{
+    display: flex;
+    flex-direction: column;
+    
+}
+.first-line-2{
+    margin-left: 0;
+}
+}
+
+/* left detaiils styling */
+.leftdetail a{
+    text-decoration: none;
+    color: black;
+    
+    display: flex;
+    align-items: center;
+   
+    
+}
+.leftdetail .img-wrapper{
+    display: flex;
+    
+}
+.leftdetail{
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+}
+@media screen and (max-width:768px) {
+    .leftdetail .img-first{
+        width: 200px;
+        
+    }
+    
+}
+/* footer */
+
+.footer{
+    background-color: #2E3192;
+    color: white;
+    display:flex;
+    justify-content: space-between;
+    padding: 30px 100px;
+}
+.footer-first .footer-second{
+    display: none;
+}
+
+.footer-second{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+}
+.playstore-wrapper img{
+    width: 100px;
+    margin-right: 20px;
+}
+.footer a{
```

Comment:
- Please remove all footer styling. the footer component already has its style.

Reply:
i have worked on every path , do well to check